Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2028,11 +2027,15 @@ - (void)didTapImageViewItem:(ConversationViewItem *)viewItem
 
     [self dismissKeyBoard];
 
-    UIWindow *window = [UIApplication sharedApplication].keyWindow;
-    CGRect convertedRect = [imageView convertRect:imageView.bounds toView:window];
-    MediaDetailViewController *vc = [[MediaDetailViewController alloc] initWithAttachmentStream:attachmentStream
-                                                                                       fromRect:convertedRect
```

Comment:
Are we not doing the presentation animations anymore?

Reply:
Yup, but they are now the responsibility of the MediaPageView controller. Which also does the dismiss animation, or a different dismiss animation if we've swiped to a different image.

Reply:
We are, but the animation responsibility has moved to the MediaPageView controller, which is also responsible for the dismiss animation. 

When we dismiss from the same media as we presented (e.g. we never swiped, or swiped and swiped back), we smoothly animate the media back to it's position in the conversation, but if we've swiped to a different media message than we started, we instead animate the media off screen.

This responsibility might shift again since I'm hoping to re-use a similar animation with the tiled media gallery.